src/mocks/keyring.mock.ts

```diff
@@ -1,23 +1,23 @@
 class KeyringMockWithInit {
-  init() {
+  static type: string;
+
+  async init() {
     return Promise.resolve();
   }
 
   getAccounts() {
     return [];
   }
 
-  serialize() {
+  async serialize() {
     return Promise.resolve({});
   }
 
-  deserialize(_) {
+  async deserialize(_) {
     return Promise.resolve();
   }
 }
 
 KeyringMockWithInit.type = 'Keyring Mock With Init';
```

Comment:
Instead of setting this outside the class, I believe you could set it inside the class like so:

``` typescript
static type = 'Keyring Mock With Init'
```

Reply:
[476124b](https://github.com/MetaMask/KeyringController/pull/202/commits/476124b7ba759f5880f79cdd5e565419473b4600)